shared/message/platform/darwin/DarwinSensors.proto

```diff
@@ -39,40 +41,37 @@ message DarwinSensors {
         CORRUPT_DATA  = 128;
         TIMEOUT       = 256;
     }
-
     message LEDPanel {
         bool led2 = 1;
         bool led3 = 2;
         bool led4 = 3;
     }
-
     message HeadLED {
-        // Encode as 0x00 0xRR 0xGG 0xBB
+        /// Encode as 0x00 0xRR 0xGG 0xBB
         uint32 RGB = 1;
     }
-
     message EyeLED {
-        // Encode as 0x00 0xRR 0xGG 0xBB
+        /// Encode as 0x00 0xRR 0xGG 0xBB
         uint32 RGB = 1;
     }
-
     message Buttons {
         bool left   = 1;
         bool middle = 2;
     }
-
+    /// In m/s^2. Each axis is bounded between -4g and 4g, according to the datasheet
```

Comment:
What data sheet?

Reply:
https://emanual.robotis.com/docs/en/platform/op2/getting_started/

Reply:
She has a point though. The datasheet should be linked in the comment